packages/@o3r/schematics/schematics/ng-add/index.ts

```diff
@@ -1,11 +1,51 @@
-import type { Rule } from '@angular-devkit/schematics';
+import type {Rule, SchematicContext, Tree} from '@angular-devkit/schematics';
+import {NodePackageInstallTask} from '@angular-devkit/schematics/tasks';
+import * as path from 'node:path';
+import {lastValueFrom} from 'rxjs';
+import {PackageJson} from 'type-fest';
+import { NodePackageName } from '@angular-devkit/schematics/tasks/package-manager/options';
 
 /**
- * Add Otter schematics to an Angular Project
+ * Install dev dependency on your application
  *
- * @param options
+ * Note: it should not be moved to other packages as it should run before the installation
+ * of peer dependencies
+ */
+class DevInstall extends NodePackageInstallTask {
+  public toConfiguration() {
+    return {
+      name: NodePackageName,
+      options: {
+        command: 'install',
+        quiet: this.quiet,
+        workingDirectory: this.workingDirectory,
+        packageName: `${this.packageName as string} --prefer-dev`,
+        packageManager: 'yarn'
+      }
+    };
+  }
+}
+
+/**
+ * Add Otter schematics to an Angular Project
  */
 export function ngAdd(): Rule {
-  /* ng add rules */
-  return () => {};
+  const schematicsDependencies = ['@angular-devkit/architect', '@angular-devkit/schematics', '@angular-devkit/core', '@schematics/angular', 'comment-json', 'eslint', 'globby', 'chokidar@^3.5.2'];
```

Comment:
Why adding `chokidar` here?
As it does not seems to be required by any `ng Add` :S.
So if a package requires it as peerDep (to its builder for example), it will be the responsibility of this specific package to add it.